content/github/searching-for-information-on-github/getting-started-with-searching-on-github/about-searching-on-github.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: About searching on GitHub
+title: About searching on GitHubgithub:main
```

Comment:
content/github/searching-for-information-on-github/getting-started-with-searching-on-github/about-searching-on-github.md